### PR TITLE
IssueID:1704:Add the source code of auto feeder.

### PR DESCRIPTION
**Pull request by wzhiyuan-nit**
[Detail]
Issue description:
#1704 Add the source code of auto feeder
Rootcause:

Solution:

Requirement description:

Solution:

[Verified Cases]
Build Pass: <solution list>
Test Pass: <test case list>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Review by ethanlcz — APPROVED** [review]
looks good to me.

File: haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/README.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+wifi_password = "请填写您的路由器密码"
+```
+
+2. **修改设备端三元组**
+
+&emsp;&emsp;
+在main.py中，填写创建的设备三元组信息。关于设备三元组的获取，请参考[创建设备及获取三元组](./README.md "创建设备及获取三元组")中的步骤。
+
+``` python
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/README.md:213]: LGFM

File: haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/code/main.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+A_ = None
+B = None
+B_ = None
+
+# 可供食用的宠物粮
+available_food = 0.0
+# 投喂次数
+feeding_times = 0
+
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/code/main.py:30]: LGFM

File: haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/code/main.py [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+productKey = "产品key"
+deviceName = "设备名称"
+deviceSecret ="设备密钥"
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/code/main.py:35]: LGFM

File: haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/README.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+&emsp;&emsp;
+将[全部代码](./code/)文件复制并覆盖刚刚创建的工程目录下的原有文件，之后对代码进行如下修改。
+
+1. **填写Wi-Fi名称及密码**
+
+&emsp;&emsp;
+在main.py中，填写可用的Wi-Fi名称及密码。
+
+``` python
+# wifi连接的的ssid和pwd定义
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/auto_feeder/haaseduk1/README.md:202]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**